### ISSUE TITLE
KeyError on com.dragonflow

### ISSUE THREAD

**Issue by ajinabraham** [JoMingyu/google-play-scraper#21]
google-play-scraper==0.0.1.1
```
from google_play_scraper import app
app('com.dragonflow')
```
```
python3.7/site-packages/google_play_scraper/__init__.py(54)app()
     53     for k, spec in ElementSpecs.Detail.items():
---> 54         content = spec.extract_content(res)
     55
ipdb>
KeyError: 'ds:15
```

**Comment by ajinabraham** [comment]
```
Python 3.7.5 (v3.7.5:5c02a39a0b, Oct 14 2019, 18:49:57)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from google_play_scraper import app
>>>
>>> result = app(
...     'com.nianticlabs.pokemongo',
...     lang='en', # defaults to 'en'
...     country='us' # defaults to 'us'
... )
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
  File "/Users/x/venv/lib/python3.7/site-packages/google_play_scraper/__init__.py", line 54, in app
    content = spec.extract_content(res)
  File "/Users/x/venv/lib/python3.7/site-packages/google_play_scraper/constants/element.py", line 20, in extract_content
    source["ds:{}".format(self.ds_num)], self.extraction_map
KeyError: 'ds:15'
``

**Comment by JoMingyu** [comment]
My response was too late, but a new patch will be released this week. sorry ;(

**Comment by JoMingyu** [comment]
Is it fixed in [0.0.1.3 release](https://pypi.org/manage/project/google-play-scraper/release/0.0.1.3/)? I'm testing about this issue.

**Comment by ajinabraham** [comment]
Yes, it's fixed.

It's a good idea to write a small test and use https://github.com/JoMingyu/google-play-scraper/actions/new to run the tests periodically to see if there is any html changes from playstore break the library.
